kuma/static/js/wiki.js

```diff
@@ -941,4 +941,9 @@
         initDetailsTags();
     }
 
+    // clear out any drafts from localStorage, now that the document has been saved
+    if (typeof document_saved !== 'undefined' && document_saved) {
+      localStorage.removeItem('draft/edit' + location.pathname);
```

Comment:
Saving a draft also sets a variable with the save-time in it. We should remove that item as well.